php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -63,15 +64,15 @@ class NDB_BVL_Feedback
      * page is receiving feedback.
      *
      * @param string  $username  The currently logged in user (why?)
-     * @param integer $candID    The 6 digit CandID
+     * @param ?CandID $candID    The 6 digit CandID
      * @param integer $sessionID The session ID with feedback
      * @param string  $commentID The commentID receiving feedback.
      *
      * @return NDB_BVL_Feedback object
      */
     static function &singleton(
         $username,
-        $candID=null,
+        ?CandID $candID=null,
```

Comment:
Why did this become nullable?

Reply:
The default param was `null` so I just kept it the same but added a type hint. I'm not sure whether there is a valid case for CandID to be null or not.

Reply:
@ridz1208 do you know if there are any valid cases for a feedback thread to have a null candid? I can't think of any, but I'm not that familiar with it.

Reply:
@ridz1208 do you know if there are any valid cases for a feedback thread to have a null candid? I can't think of any, but I'm not that familiar with it.

Reply:
logically no, the highest place you can put a feedback is at the candidate level... so no

Reply:
I'm okai with this given that it maintains status quo however if should be removed in the future